### ISSUE TITLE
check refactor edges

### ISSUE THREAD

**Issue by kknierim-usgs** [brclark-usgs/NHGtools#5]
When the national grid is resized to a different factor (or resolution), may want to check the behavior of the grid at edges. It may be performing as we expect, but I want to double check with some examples. @brclark-usgs  I'll work on this too, wanted to set up an issue to track.

**Comment by kknierim-usgs** [comment]
After looking at NHG.py, I think the extent is being purposefully reset (line 140)
`syoff = (self.ext['ll'][1] - self.__natlExt['ll'][1]) / res`
I'm guessing that is on purpose, but I can't see why. @brclark-usgs shouldn't the extent be the same?

**Comment by brclark-usgs** [comment]
so i'm thinking it's creating a grid that is closest to the user input extents and still aligns with the National grid. So if it's 1/10 of the 1km grid, it will still align, but it can begin at a point within a 1km cell. does that make sense?

**Comment by kknierim-usgs** [comment]
right, but if it's some fraction of the national grid--it can align and still have the same extent, right? that's not mutually exclusive. So you're probably thinking one step ahead of me, for grids (for example 30 m) that we would want to align as much as possible? If we just think about fractions... we could always keep alignment and extent (i think)
